samples/default-authorizationserver/src/test/java/sample/DefaultAuthorizationServerConsentTests.java

```diff
@@ -58,15 +62,18 @@ public class DefaultAuthorizationServerConsentTests {
 	@MockBean
 	private OAuth2AuthorizationConsentService authorizationConsentService;
 
-	private final String redirectUri = "http://127.0.0.1/login/oauth2/code/messaging-client-oidc";
+	private final String redirectUri = "http://127.0.0.1:8080/login/oauth2/code/messaging-client-oidc?param=is%2Fencoded";
```

Comment:
Please revert all changes in `DefaultAuthorizationServerConsentTests` since we have same test coverage in `DefaultAuthorizationServerApplicationTests`